std/fs/empty_dir.ts

```diff
@@ -1,24 +1,41 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { join } from "../path/mod.ts";
+const {
+  readDir,
+  readDirSync,
+  mkdir,
+  mkdirSync,
+  remove,
+  removeSync,
+  ErrorKind
+} = Deno;
 /**
  * Ensures that a directory is empty.
  * Deletes directory contents if the directory is not empty.
  * If the directory does not exist, it is created.
  * The directory itself is not deleted.
+ * Requires the `--allow-read` and `--alow-write` flag.
  */
 export async function emptyDir(dir: string): Promise<void> {
   let items: Deno.FileInfo[] = [];
   try {
-    items = await Deno.readDir(dir);
-  } catch {
+    items = await readDir(dir);
+  } catch (err) {
+    if ((err as Deno.DenoError<Deno.ErrorKind>).kind !== ErrorKind.NotFound) {
```

Comment:
This can be shortened because ErrorKind is in scope

Reply:
This will cause compilation errors

```
error TS2749: 'ErrorKind' refers to a value, but is being used as a type here.
```

see [ci log](https://github.com/denoland/deno/pull/3501/checks?check_run_id=349225487#step:20:1030)